src/main/java/hudson/plugins/sonar/MsBuildSQRunnerInstallation.java

```diff
@@ -40,19 +40,23 @@
 import net.sf.json.JSONObject;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.StaplerRequest;
+import sun.launcher.resources.launcher;
 
 public class MsBuildSQRunnerInstallation extends ToolInstallation implements EnvironmentSpecific<MsBuildSQRunnerInstallation>, NodeSpecific<MsBuildSQRunnerInstallation> {
   private static final long serialVersionUID = 1L;
-  private static String exeName = "SonarScanner.MSBuild.exe";
+  private static final String SCANNER_EXE_NAME = "SonarScanner.MSBuild.exe";
+  private static final String SCANNER_DLL_NAME = "SonarScanner.MSBuild.dll";
+  private static final String OLD_SCANNER_EXE_NAME = "MSBuild.SonarQube.Runner.exe";
+  private static String testExeName = null;
```

Comment:
I haven't dig too much about the why but it doesn't work when this is not `static`.

Reply:
ok...